.travis.yml

```diff
@@ -1,15 +1,3 @@
 language: java
-jdk:
-  - oraclejdk8
-# A peculiarity of dependency caching in Gradle means that to avoid uploading the cache after every build the following
-# lines must be added:
-before_cache:
```

Comment:
Clearing gradle caches is still a recommended thing: https://docs.travis-ci.com/user/languages/groovy/#gradle-caching

So, please leave `before_cache` and `cache` keys.

Reply:
(I put back whose configuration lines)